tests/automation/release_notes/test_generate.py

```diff
@@ -81,3 +86,94 @@ def test_commit_regex_matching():
+
+* **UI**: [Features & enhancement](https://github.com/mlrun/ui/releases/tag/{release}#features-and-enhancements)
+
+### Bug fixes
+* **UI**: [Bug fixes](https://github.com/mlrun/ui/releases/tag/{release}#bug-fixes)
+
+
+#### Pull requests:
+fd6c5a86 [Requirements] Bump storey to 0.8.15 and v3io-frames to 0.10.2 (#1553)
+985d7cb8 [Secrets] Verify project secrets do not exist when deleting a project (#1552)
+
+#### Failed parsing:
+
+        \n""",
+        },
+        {
+            "_run_command": [
+                None,
+                "fd6c5a86 {Gal Topper} [Requirements] Bump storey to 0.8.15 and v3io-frames to 0.10.2 (#1553)\n"
+                "20d4088c {yuribros1974} Merge pull request #1511 from mlrun/ML-509_update_release_status\n"
+                "985d7cb8 {Saar Cohen} [Secrets] Verify project secrets do not exist when deleting a project (#1552)",
+                "fd6c5a86 [Requirements] Bump storey to 0.8.15 and v3io-frames to 0.10.2 (#1553)\n"
+                "20d4088c Merge pull request #1511 from mlrun/ML-509_update_release_status\n"
+                "985d7cb8 [Secrets] Verify project secrets do not exist when deleting a project (#1552)",
+            ],
+            "_resolve_github_username": ["gtopper", "theSaarco"],
+            "expected_response": f"""
+### Features / Enhancements
+* **Requirements**: Bump storey to 0.8.15 and v3io-frames to 0.10.2, #1553, @gtopper
+* **Secrets**: Verify project secrets do not exist when deleting a project, #1552, @theSaarco
+
+* **UI**: [Features & enhancement](https://github.com/mlrun/ui/releases/tag/{release}#features-and-enhancements)
+
+### Bug fixes
+* **UI**: [Bug fixes](https://github.com/mlrun/ui/releases/tag/{release}#bug-fixes)
+
+
+#### Pull requests:
+fd6c5a86 [Requirements] Bump storey to 0.8.15 and v3io-frames to 0.10.2 (#1553)
+20d4088c Merge pull request #1511 from mlrun/ML-509_update_release_status
+985d7cb8 [Secrets] Verify project secrets do not exist when deleting a project (#1552)
+
+#### Failed parsing:
+20d4088c {{yuribros1974}} Merge pull request #1511 from mlrun/ML-509_update_release_status
+        \n""",
+        },
+    ]
+    automation.release_notes.generate.tempfile = mock.MagicMock()
+    for case in cases:
+        with mock.patch(
+            "automation.release_notes.generate.ReleaseNotesGenerator._run_command"
+        ) as _run_command_mock, mock.patch(
+            "automation.release_notes.generate.ReleaseNotesGenerator._resolve_github_username"
+        ) as _resolve_github_user_mock, mock.patch(
+            "sys.stdout", new=StringIO()
+        ) as fake_out:
+            _run_command_mock.side_effect = case["_run_command"]
+            _resolve_github_user_mock.side_effect = case["_resolve_github_username"]
+            try:
+                release_generator.run()
+            except ValueError:
+                logger.warning(
+                    "Test case had some commits that didn't pass the requirements"
+                )
```

Comment:
Change the code here to do
```suggestion
            except ValueError:
                if not case.get("expect_failure", False):
                    raise
```
And then in the cases where failure is expected, add a field `"except_failure": True`